vcsclient/azurerepos.go

```diff
@@ -177,6 +177,30 @@ func (client *AzureReposClient) CreatePullRequest(ctx context.Context, _, reposi
 	return err
 }
 
+// UpdatePullRequest on Azure Repos
+func (client *AzureReposClient) UpdatePullRequest(ctx context.Context, owner, repository, title, body, targetBranchName string, prId int, state vcsutils.PullRequestState) error {
+	azureReposGitClient, err := client.buildAzureReposClient(ctx)
+	if err != nil {
+		return err
+	}
+	if targetBranchName != "" {
```

Comment:
The `vcsutils.AddBranchPrefix(targetBranchName)` won't fail in case of empty string